user/client/client.go

```
@@ -70,13 +71,24 @@ func (c *Client) Get(ctx context.Context, id string) (*user.User, error) {
+	if err != nil {
 		if request.IsErrorResourceNotFound(err) {
 			return nil, nil
 		}
 		return nil, err
 	}
 
+	result := &user.User{}
+	if body == nil {
+		return result, nil
+	}
+	defer body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `body.Close` is not checked

(errcheck)